test/run/issue-base653.mo

```diff
@@ -0,0 +1,3 @@
+import { floatToInt64 } = "mo:prim";
+
+floatToInt64 (-0.5);
```

Comment:
Should we maybe also test `floatToInt64(floatCopySign(0.0, -1.0)))`?

Reply:
Done: 5f9f320a0